src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -428,7 +428,11 @@
           [WorkspaceTable
            (merge props (utils/restructure workspaces featured-workspaces))])
         {:loading-text "Loading workspaces..."
-         :rephrase-error #(get-in % [:parsed-response :workspaces :error-message])})))
+         :rephrase-error (fn [ajax-response]
+                           (let [msg (or
+                                      (get-in ajax-response [:parsed-response :message])
+                                      (get-in ajax-response [:parsed-response :workspaces :error-message]))]
+                            [:div {:style {:padding "50px"}} msg]))})))
```

Comment:
from reverse-engineering the backend code, I don't see anything that delivers an error message in `[:parsed-response :workspaces :error-message]`. But, it felt safe enough to leave that path in, as the second clause in the `(or)` statement.